web/modules/custom/user_timezone/src/UserTimezoneSalutation.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\user_timezone;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Prepares user greeting.
+ */
+class UserTimezoneSalutation extends ControllerBase {
```

Comment:
Is this a controller or a service? I think this is a service, then it shouldn't inherit from `ControllerBase`